discovery-provider/plugins/notifications/src/processNotifications/indexAppNotifications.ts

```diff
@@ -123,8 +133,11 @@ export class AppNotificationsProcessor {
       )
       if (isEnabled) {
         const isLiveEmailEnabled = this.getIsLiveEmailEnabled()
+        const isBrowserPushEnabled = this.getIsBrowserPushEnabled()
+        logger.info(`live email ${isLiveEmailEnabled ? "is" : "is not"} enabled`)
```

Comment:
are these logs intentional for every notification? maybe move the gets outside too.

Reply:
good catch